File: csharp/bypstest-ser/src-ser/byps/test/api/remote/RemoteReferences.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using byps;

namespace byps.test.api.remote
{
	
	// checkpoint byps.gen.cs.GenRemoteClass:93
	public interface RemoteReferences : BRemote {
	
		byps.test.api.refs.Node GetNode();
		void GetNode(BAsyncResult<byps.test.api.refs.Node> asyncResult) ;
		Task<byps.test.api.refs.Node> GetNodeAsync();
		
		void SetNode(byps.test.api.refs.Node v);
		void SetNode(byps.test.api.refs.Node v, BAsyncResult<Object> asyncResult) ;
		Task SetNodeAsync(byps.test.api.refs.Node v);
		
		
	}
}  // end namespace

File: java/bypstest-api/src/byps/test/api/remote/RemoteReferences.java
package byps.test.api.remote;

import byps.BRemote;
import byps.RemoteException;
import byps.test.api.refs.Node;

public interface RemoteReferences extends BRemote {
	Node getNode() throws RemoteException;
	void setNode(Node v) throws RemoteException;
}
